bundles/org.openhab.binding.weatherunderground/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -417,6 +465,10 @@
 		<label>Rain Fall Day</label>
 		<description>Rain fall during the day</description>
 		<category>Rain</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Rain</tag>
+		</tags>
```

Comment:
Why is it "Measurement" for this channel type while it is "Forecast" for all other ?

Reply:
Your choice. Measurement indicates the verb present tense, and Forecast indicates the future. The actual Measurement / Forecast can come from a local physical device or from a remote weather service (which obviously has their own physical devices around the country).